lib/cloudfunc.js

```diff
@@ -218,6 +218,16 @@
                     mode        : '--- --- ---'
                 });
             }
+
+            // OSC_CUSTOM_CODE hide all dot files
+            n = files.length;
+            var filesWithoutDots = [];
+            for (i = 0; i < n; i++) {
+                if (files[i].name.charAt(0) !== '.') {
+                    filesWithoutDots.push(files[i]);
+                }
+            }
+            files = filesWithoutDots;
```

Comment:
This is being performed client-side based on a json object, so arrow functions won't be supported in IE, Safari, or Opera.

http://caniuse.com/#feat=arrow-functions